apps/authentication-portal/src/main/webapp/org_name.jsp

```diff
@@ -0,0 +1,155 @@
+<%--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The license header should be updated in the new format
```
Copyright (c) $today.year, WSO2 Inc. ([http://www.wso2.c](http://www.wso2.com/)om).

WSO2 Inc. licenses this file to you under the Apache License,
Version 2.0 (the "License"); you may not use this file except
in compliance with the License.
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
```

Reply:
Fixed.